src/app.js

```diff
@@ -25,11 +39,61 @@ const quoteData = [
   },
 ];
 
+// Got quote names & prices into separate arrays
+const names = quoteData.map(info => info.symbol);
+const prices = quoteData.map(info => info.price);
+
+// Define some variables
+let quoteTemplate;
+let tradeTemplate;
+let orderTemplate;
+
+// -------------------------------------------------------
+
+// jQuery Ready
 $(document).ready(function() {
+
+  // Event Bus
+  let bus = {};
+  bus = _.extend(bus, Backbone.Events);
+
+  // Templates
+  quoteTemplate = _.template($('#quote-template').html());
+  tradeTemplate = _.template($('#trade-template').html());
+  orderTemplate = _.template($('#order-template').html());
+
+  // Constants
   const quotes = new QuoteList(quoteData);
   const simulator = new Simulator({
     quotes: quotes,
   });
+  const orders = new OrderList();
+
+  // Render quote list view
+  const quoteListView = new QuoteListView({
+    el: $('#quotes-container'),
+    model: quotes,
+    template: quoteTemplate,
+    bus: bus,
+  });
+
+  // Render trade history view
+  const tradeView = new TradeView({
+    el: $('#trades-container'),
+    template: tradeTemplate,
+    bus: bus,
+  })
+
+  // Render order list view
+  const orderListView = new OrderListView({
+    el: $('#order-workspace'),
+    model: orders,
+    template: orderTemplate,
+    quoteList: quotes,
```

Comment:
I like that the `OrderListView` knows about the `QuoteList`. I've seen several students attempt to achieve the same thing using the bus as an intermediary, but the code ends up being much more complex.